### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by Jakuje**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Test suite updated with negative tests
+- [ ] Documentation updated
+
+
+#### Reviewer's checklist:
+
+- [ ] Any issues marked for closing are addressed
+- [ ] There is a test suite reasonably covering new functionality or modifications
+- [ ] This feature/change has adequate documentation added
+- [ ] No obvious mistakes in the code
```
Comment (simo5) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: I would drop this one and instead have:
- [ ] Code conform to coding style that today cannot yet be enforced via the check style test

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#### Description
+
```
Comment (simo5) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Add:
The description should give a general overview of the goal of the PR.
Individual commit message should highlight important changes that people may want to know about year later when sorting thorough as commits are the changelog.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (simo5) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: Also add
- [ ] Commits have short titles and sensible commit messages

**Review by simo5 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Jakuje) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: Done.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Jakuje) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Review by simo5 — APPROVED** [review]

**Pull request merged**